lib/src/base-styles/_nx-radio-checkbox.scss

```diff
@@ -31,9 +32,15 @@
   }
 
   &:focus-within {
+    &::after {
+      border: 1px solid var(--nx-color-interactive-border-focus);
+      box-shadow: var(--nx-box-shadow-focus);
+      content: '';
+      display: block;
+      position: absolute;
+    }
+
     .nx-radio-checkbox__control {
-      border-color: var(--nx-color-interactive-border-focus);
-      stroke: var(--nx-color-interactive-border-focus);
       filter: drop-shadow(var(--nx-drop-shadow-focus));
```

Comment:
Shouldn't this be removed? It creates an additional glow around the radio/checkbox itself.

Reply:
> Shouldn't this be removed? It creates an additional glow around the radio/checkbox itself.

It has been removed.
37849b5e287821b39691cdefdf4c1a6165f13bfb